cmd/chaos-builder/main.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 var (
-	log = zap.Logger(true)
+	log = zap.New(zap.UseDevMode(true))
```

Comment:
Does `UseDevMode=true` have any impact on performance?

Reply:
No, there is no performance improvement, only standardize the way it is used.